Source/Scene/Scene.js

```diff
@@ -1310,6 +1456,15 @@ define([
 
             scene._globeDepth.executeCopyDepth(context, passState);
 
+            if (scene.debugShowGlobeDepth) {
```

Comment:
Yeap - I think this will still be quite clean compared to the manual frambuffers/textures and copying.